tests/test_util.py

```diff
@@ -319,7 +321,7 @@ def test_axis_sort_badndim(ndim, axis, index, value):
 @pytest.mark.parametrize("value", [None, np.min, np.mean, np.max])
 def test_axis_sort(ndim, axis, index, value):
     srand()
-    data = np.random.randn(*([10] * ndim))
+    data = cast(np.ndarray, np.random.randn(*([10] * ndim)))
```

Comment:
Why do we need these casts?  seems a bit weird, since np.random.randn should always return an ndarray.

Reply:
randn can also return a float if no parameters are supplied, and here since we use the star operator the inferred type is `float | ndarray`

Reply:
Seems like it would be cleaner to use `np.asarray(np.random.randn(...))` instead of a cast?

Reply:
ok done